deepchem/models/torch_models/cgcnn.py

```diff
@@ -26,11 +26,11 @@ class CGCNNLayer(nn.Module):
   41
   >>> cgcnn_dgl_graph = cgcnn_graph.to_dgl_graph()
   >>> print(type(cgcnn_dgl_graph))
-  <class 'dgl.graph.DGLGraph'>
+  <class 'dgl.heterograph.DGLHeteroGraph'>
```

Comment:
The fix is related to the new version release. (DGL. 0.5.0 released today). I seem the master branch CI also failed.